src/main/java/com/google/gerrit/extensions/api/groups/Groups.java

```diff
@@ -85,12 +85,12 @@
 
     public List<GroupInfo> get() throws RestApiException {
       Map<String, GroupInfo> map = getAsMap();
-      List<GroupInfo> result = new ArrayList<>(map.size());
-      for (Map.Entry<String, GroupInfo> e : map.entrySet()) {
-        // ListGroups "helpfully" nulls out names when converting to a map.
-        e.getValue().name = e.getKey();
-        result.add(e.getValue());
-      }
+      List<GroupInfo> result = new ArrayList<>(map.values());
```

Comment:
Files in the com.google package should not be touched as these are copied over from official Gerrit source from time to time. Does your "fix" also work without this change? Or do you see any alternatives to fix this issue?